packages/security/libressl/package.mk

```diff
@@ -17,23 +17,28 @@
 ################################################################################
 
 PKG_NAME="libressl"
-PKG_VERSION="2.3.8"
+PKG_VERSION="2.4.3"
 PKG_REV="1"
 PKG_ARCH="any"
 PKG_LICENSE="BSD"
 PKG_SITE="http://www.libressl.org/"
 PKG_URL="http://ftp.openbsd.org/pub/OpenBSD/LibreSSL/$PKG_NAME-$PKG_VERSION.tar.gz"
+PKG_DEPENDS_HOST="ccache:host"
 PKG_DEPENDS_TARGET="toolchain"
+PKG_PRIORITY="optional"
 PKG_SECTION="security"
 PKG_SHORTDESC="libressl: a FREE version of the SSL/TLS protocol forked from OpenSSL"
 PKG_LONGDESC="LibreSSL is a FREE version of the SSL/TLS protocol forked from OpenSSL"
 
 PKG_IS_ADDON="no"
-PKG_USE_CMAKE="no"
-PKG_AUTORECONF="yes"
+PKG_AUTORECONF="no"
 
 post_makeinstall_target() {
-  # backwards comatibility
   mkdir -p $INSTALL/etc/pki/tls
     ln -sf /etc/ssl/cert.pem $INSTALL/etc/pki/tls/cacert.pem
+  mkdir -p $INSTALL/etc/pki/tls/certs
+    ln -sf /etc/ssl/cert.pem $INSTALL/etc/pki/tls/certs/ca-bundle.crt
+
+  mkdir -p $SYSROOT_PREFIX/usr/lib/pkgconfig
+    cp $PKG_DIR/config/*.pc $SYSROOT_PREFIX/usr/lib/pkgconfig
```

Comment:
If you change the hard-coded version in *.pc from `2.4.3` to `@PKG_VERSION@`, eg:
```
Version: @PKG_VERSION@
```
then use a `sed` to fix the *.pc files, ie. 
```
    sed -e "s/@PKG_VERSION@/$PKG_VERSION/g" -i $SYSROOT_PREFIX/usr/lib/pkgconfig/*.pc
```
and then we don't need to keep all the files in sync.

Not sure if applying the sed to all files in $SYSROOT_PREFIX/usr/lib/pkgconfig might produce unexpected results, in which case we should process the individual source files and output the new content to $SYSROOT_PREFIX/usr/lib/pkgconfig, ie:
```
     for f in $PKG_DIR/config/*.pc; do
       sed -e "s/@PKG_VERSION@/$PKG_VERSION/g" $f > $SYSROOT_PREFIX/usr/lib/pkgconfig/$(basename $f)
     done
```